### PR TITLE
5.6.10 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.6.md [hunk 1]
```diff
@@ -28,6 +28,32 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+### New Features
+
+- **I95-52198 Handle incoming public keys from peer conductor node:** Added functionality to allow conductor nodes to share the authorized keys of managed routers between each other. If the SSH public key is retrieved from a managed router by one conductor node then it will be automatically shared with it's conductor peer node.
+
+### Resolved Issues
+
+- **The following CVE's have been identified and addressed in this release:** I95-51758, I95-52495, I95-52497, I95-52509.
+------
+- **I95-51336 App-ID stats entry leaked in some session building exceptions:** Resolved an issue where the `app-id stats` entry was not added to the `Expiring` list to be cleaned up.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:43]: While the Jira talks about "stats entries are being leaked", this really amount to a memory leak.
**I95-51336 App-ID memory leak for some uncommon cases, such a duplicate flow:** Resolved an issue where the `app-id stats` entry was not added to the `Expiring` list to be cleaned up.

File: docs/release_notes_128t_5.6.md [hunk 2]
```diff
@@ -28,6 +28,32 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-51800 Radius authentication failure - Incorrect NAS IP address:** The ability to specify the NAS-IP-Address and NAS-Identifier has been added to the data model for configuring these Radius options per node. This can be used in cases where the Radius server is configured to use an identifier, or in cases where it is necessary to match the source IP address of the Radius requests behind SSR or NAT.
+------
+- **I9552208 Queries return incomplete data with FIPS enabled:** Resolved an issue where a FIPS-incompatible hashing function was causing missing or incomplete metrics data.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:47]: What queries? Also, missing hyphen
**I95-52208 metrics queries return incomplete data when FIPS is enabled: Resolved an issue where a FIPS-incompatible hashing function was causing missing or incomplete metrics data.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_5.6.md [hunk 3]
```diff
@@ -28,17 +28,19 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
 ### New Features
 
 - **I95-52198 Handle incoming public keys from peer conductor node:** Added functionality to allow conductor nodes to share the authorized keys of managed routers between each other. If the SSH public key is retrieved from a managed router by one conductor node then it will be automatically shared with it's conductor peer node.
 
 ### Resolved Issues
 
-- **The following CVE's have been identified and addressed in this release:** I95-51758, I95-52495, I95-52497, I95-52509.
+- **The following CVE's have been identified and addressed in this release:** I95-51758, I95-52495, I95-52496, I95-52497, I95-52509, I95-52625. 
+------
+- **I95-41386/I95-52114 HA pair device interface's redundancy status stays non-redundant even though the interface operational status is up:** Resolved an issue where zookeepper enters a leaderless state when a disconnection occurs right after an electorate ephemeral node is created, and before running a success callback.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:43]: Suggested rewording
- **I95-41386/I95-52114 HA pair device interface's redundancy status stays non-redundant even though the interface operational status is up after both nodes simultaneously come online, as in the case of power failures:** Resolved a race condition where zookeepper enters a leaderless state when a disconnection occurs right after an electorate ephemeral node is created, and before running a success callback.

File: docs/release_notes_128t_5.6.md [hunk 4]
```diff
@@ -51,7 +53,11 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
 - **I95-52305 Compacting rate limit exceeded:** Resolved memory and CPU issues resulting from attempting to compact very large application identification documents.
 ------
 - **I95-52402 Router stuck in `Upgrading` state:** Resolved an issue with `conductor-only` mode, where the conductor was attempting to download the installer before the software access proxies were in place, preventing an update to the installer.
-------  
+------
+- **I95-52626 Forwarding plane control message bursts create exception, causing a packet buffer leak:** Resolved a condition where backpressure from fastlane caused the messaging mechanism between highway manager and fastlane to drop mbufs. Proper handling of exception now prevents buffer leaks. Additionally, increased the control buffer capacity to better handle bursts as part of the resolution.
+------
+- **I95-52650 Create a cache of the common file root hash calculation:** An optimization was made to an internal calculation and improve the speed at which synchronization requests are processed.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.6.md:59]: Suggested rewording:

- **I95-52650 Asset state transition on conductor is slow for deployments with greater than 250 routers:** An optimization was made to an internal calculation and improve the speed at which synchronization requests are processed.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.6.md (hunk 3 above)
Comment (hadrielk) [reply docs/release_notes_128t_5.6.md:43]: 1. "zookeeper" is misspelled as "zookeepper" in both the original and your rewording.
2. Do these internal details about zookeeper and such really mean anything to customers?

File: docs/release_notes_128t_5.6.md [hunk 5]
```diff
@@ -28,6 +28,38 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 The Juniper SSR team does not publicly disclose known or resolved CVEs in our public documentation but instead utilizes our internal bug tracking IDs. Customers can obtain the actual CVE numbers by contacting Juniper Support.
 :::
 
+## Release 5.6.10-5
+
+**Release Date:** August 31, 2023
+
+### New Features
+
+- **I95-52198 Handle incoming public keys from peer conductor node:** Added functionality to allow conductor nodes to share the authorized keys of managed routers between each other. If the SSH public key is retrieved from a managed router by one conductor node then it will be automatically shared with it's conductor peer node.
```
Comment (hadrielk) [inline docs/release_notes_128t_5.6.md:37]: the word "it's" at the end of the paragraph should be "its".

File: docs/release_notes_128t_5.6.md [hunk 6]
```diff
@@ -28,6 +28,38 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-52283 Correct the Domain Matching order:** When using web filtering, the SSR now properly enforces the [Service Matching Order.](https://www.juniper.net/documentation/us/en/software/session-smart-router/docs/config_domain-based_web_filter/#service-matching-order)
+------
+- **I95-52305 Compacting rate limit exceeded:** Resolved memory and CPU issues resulting from attempting to compact very large application identification documents.
+------
+- **I95-52402 Router stuck in `Upgrading` state:** Resolved an issue with `conductor-only` mode, where the conductor was attempting to download the installer before the software access proxies were in place, preventing an update to the installer.
+------
+- **I95-52626 Forwarding plane control message bursts create exception, causing a packet buffer leak:** Resolved a condition where backpressure from fastlane caused the messaging mechanism between highway manager and fastlane to drop mbufs. Proper handling of exception now prevents buffer leaks. Additionally, increased the control buffer capacity to better handle bursts as part of the resolution.
```
Comment (hadrielk) [inline docs/release_notes_128t_5.6.md:57]: again, I'm not sure we should be documenting this level of internal details, which are not meaningful to customers.

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — CHANGES_REQUESTED** [review]
Still need to make requested changes.

File: docs/release_notes_128t_5.6.md [hunk 7]
```diff
@@ -28,6 +28,44 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+- **I95-52198 Handle incoming public keys from peer conductor node:** Added functionality to allow conductor nodes to share the authorized keys of managed routers between each other. If the SSH public key is retrieved from a managed router by one conductor node, then it is automatically shared with its conductor peer node.
+------
+- **I95-52517 Allow users the ability to configure the OSPF SPF timers:** Support for user-configured values for SPF delay has been added. Users can now specify values for spf delay, hold-time, and maximum-hold-time. For additional information, see [OSPF SPF Timers](config_command_guide.md#configure-authority-router-routing-ospf-timers-spf).
+
+### Resolved Issues
+
+- **The following CVE's have been identified and addressed in this release:** I95-51758, I95-52495, I95-52496, I95-52497, I95-52509, I95-52625. 
+------
+- **I95-41386/I95-52114 HA pair device interface's redundancy status stays non-redundant even though the interface operational status is up:** Resolved a race condition where a disconnection occurs right after an electorate ephemeral node is created, and before running a success callback.
```
Comment (hadrielk) [inline docs/release_notes_128t_5.6.md:45]: This resolution isn't meaningful to customers; I suggest it be:

"Resolved a race condition when selecting the active components between HA nodes."

File: docs/release_notes_128t_5.6.md [hunk 8]
```diff
@@ -28,6 +28,44 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-51800 Radius authentication failure - Incorrect NAS IP address:** The ability to specify the NAS-IP-Address and NAS-Identifier has been added to the data model for configuring these Radius options per node. This can be used in cases where the Radius server is configured to use an identifier, or in cases where it is necessary to match the source IP address of the Radius requests behind SSR or NAT.
+------
+- **I95-52208 Metrics queries return incomplete data when FIPS is enabled:** Resolved an issue where a FIPS-incompatible hashing function was causing missing or incomplete metrics data. 
+------
+- **I95-52283 Correct the Domain Matching order:** When using web filtering, the SSR now properly enforces the [Service Matching Order.](config_domain-based_web_filter.md/#service-matching-order)
+------
+- **I95-52305 Compacting rate limit exceeded:** Resolved memory and CPU issues resulting from attempting to compact very large application identification documents.
+------
+- **I95-52316 Enhancements to Overlapping FIB Services:** When creating FIB entries, we now consider services that match the route update but do not have the best match service address.
```
Comment (hadrielk) [inline docs/release_notes_128t_5.6.md:57]: I believe this requires setting an authority-wide configuration knob, no?

File: docs/release_notes_128t_5.6.md [hunk 9]
```diff
@@ -28,6 +28,44 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-52402 Router stuck in `Upgrading` state:** Resolved an issue with `conductor-only` mode, where the conductor was attempting to download the installer before the software access proxies were in place, preventing an update to the installer.
+------
+- **I95-52626 Forwarding plane control message bursts create exception, causing a packet buffer leak:** Resolved a condition where backpressure caused the messaging mechanism to develop buffer leaks. Proper handling of exceptions now prevents buffer leaks. Additionally, increased the control buffer capacity to better handle bursts as part of the resolution.
```
Comment (hadrielk) [inline docs/release_notes_128t_5.6.md:61]: I would remove the "Additionally, ..." last sentence.

**Review by Bhautik — APPROVED** [review]

**Review by hadrielk — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**